server/portal/apps/tickets/api/views.py

```diff
@@ -46,7 +46,7 @@ def post(self, request):
         data = request.POST.copy()
         subject = data.get('subject')
         problem_description = data.get('problem_description')
-        cc = data.get('cc', '')
+        cc = data.get('cc', [''])
```

Comment:
should this be `['']` or `[]`? In another change in this PR, the default is `[]`

Reply:
I'll revert it to `[]`.